src/main/java/edu/cornell/kfs/vnd/businessobject/options/EinvoiceIndicatorValuesFinder.java

```diff
@@ -0,0 +1,51 @@
+package edu.cornell.kfs.vnd.businessobject.options;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.krad.keyvalues.KeyValuesBase;
+import org.kuali.rice.core.api.util.ConcreteKeyValue;
+import org.kuali.rice.core.api.util.KeyValue;
+
+public class EinvoiceIndicatorValuesFinder extends KeyValuesBase {
+    private static final long serialVersionUID = 6328041347235067740L;
+    private static final Logger LOG = LogManager.getLogger();
+
+    @Override
+    public List<KeyValue> getKeyValues() {
+        LOG.debug("getKeyValues, entering");
```

Comment:
I typically put my log statements as function name comma whatever is needed ..... actually the logging was only needed was I was debugging  an odd issue, I think it would make sense just to remove all the log stuff from this class